src/test/scala/com/microsoft/hyperspace/index/IndexLogEntryTest.scala

```diff
@@ -439,6 +439,34 @@ class IndexLogEntryTest extends SparkFunSuite with SQLHelper with BeforeAndAfter
     FileUtils.deleteDirectory(tempDir.toFile)
   }
 
+  test("Directory.merge Test: test if merge works as expected.") {
+    // File Structure
+    // testDir/a/f1
+    // testDir/a/f2
+    // testDir/a/b/f3
+    // testDir/a/b/f4
```

Comment:
Since you wrote this down, can you also write down the expected result?

Reply:
added, thanks @rapoth

Reply:
Thanks @imback82 , I updated these tests to be hardcoded and followed the instructions here.